lua/entities/gmod_wire_expression2/core/number.lua

```diff
@@ -226,6 +228,27 @@ end)
 
 __e2setcost(2) -- approximation
 
+--- Returns true (1) if given value is a finite number; otherwise false (0).
+e2function number finite(value)
+	return value > -huge and value < huge
```

Comment:
@TomyLobo Why...? That would be a redundant check, as it already returns false (0) if given NaN. Have you seen [unit tests in pure Lua](https://gist.github.com/CaptainPRICE/342303bba104354257db0471bf996bd3)? (I have posted a link to that on issue page.) See "Finite 3" test - passed.  :)